src/index.html

```diff
@@ -11,6 +11,46 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="form"
+      action="/search"
+      method="get"
+      data-qa="big"
+    >
+      <img
+        class="search-icon"
+        src="./images/Search.svg"
+        alt="search image"
+      >
+      <label for="input">
```

Comment:
incorrect for attribute
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label